internal/log/syslog.go

```diff
@@ -60,6 +61,11 @@ func (h *syslogHandler) HandleLog(e *apexLog.Entry) error {
 	enc := json.NewEncoder(&bld)
 	enc.Encode(syslogFields(e))
 
+	if h.writer == nil {
+		stdlog.Printf("Error logging: Syslog unavailable, please change the logging config")
```

Comment:
I guess this is going to stdout, can you please make it go to stderr instead?

Reply:
Yes, will change to
```
        if h.writer == nil {
-               stdlog.Printf("Error logging: Syslog unavailable, please change the logging config")
+               fmt.Fprintf(os.Stderr, "Error logging: Syslog unavailable, please change the logging config")
                return nil
        }
```